scripts/thumbnail-generator/generate-thumbnail.js

```diff
@@ -0,0 +1,55 @@
+const { chromium } = require('playwright');
+const { expect } = require('@playwright/test');
+const sharp = require('sharp');
+const fs = require('fs');
+const path = require('path');
+
+(async () => {
+  const browser = await chromium.launch();
+  const context = await browser.newContext({
+    viewport: {
+      width: 1280, // Set the desired window width here
+      height: 720, // Set the desired window height here
+    },
+  });
+  const page = await context.newPage();
+
+  await context.addCookies([
```

Comment:
Potentially we could inpersonate a user to get a thumbnail of their private data, it seems that this is not needed for the first iteration